Units/aiunits/eaiturret.lua

```diff
@@ -57,8 +57,8 @@ local unitDef                    = {
 	leaveTracks                  = false,
 	maxDamage                    = maxDamage,
 	maxSlope                     = 26,
-	maxVelocity                  = 0.001,
-	maxReverseVelocity           = 0.001,
+	maxVelocity                  = 1,

```

Comment:
I wanted it to be turret but to move when AI is trying to place something on it..